Update supported devices and VISA backends in the readme (#170)
tektronix/tm_devices: README.rst

`PyVISA-py <https://pyvisa.readthedocs.io/projects/pyvisa-py/en/latest/>`__
VISA backend. However, in order to use any VISA functionality that
PyVISA-py does not implement, a third-party VISA backend such as

> `TekVISA <https://www.tek.com/en/search?keywords=tekvisa&facets=_templatename%3dsoftware%26parsedsoftwaretype%3dDriver&sort=>`__

Reviewer: This section is not valid syntax for rST. Please verify that the docs build locally.

Author: Run and Verified
![image](https://github.com/tektronix/tm_devices/assets/3426243/a701336f-9e3f-4ed1-9c7f-73017fab05ea)

`PyVISA-py <https://pyvisa.readthedocs.io/projects/pyvisa-py/en/latest/>`__
VISA backend. However, in order to use any VISA functionality that
PyVISA-py does not implement, a third-party VISA backend such as

> `TekVISA <https://www.tek.com/en/search?keywords=tekvisa&facets=_templatename%3dsoftware%26parsedsoftwaretype%3dDriver&sort=>`__

Reviewer: ```suggestion
`TekVISA <https://www.tek.com/en/search?keywords=tekvisa&facets=_templatename%3dsoftware%26parsedsoftwaretype%3dDriver&sort=>`__ (>=4.2.0)
```

or
`NI-VISA <https://www.ni.com/en-us/support/downloads/drivers/download.ni-visa.html>`__

> can be installed. v4.2.0 is the TekVISA version supported by ``tm_devices``.

Reviewer: The minimum version for TekVISA needs to moved above to be next to the TekVISA link

Reviewer: ```suggestion
can be installed. ``tm_devices`` will use an available VISA backend if
```

Author: Done.

> ``tm_devices`` will use an available VISA backend if

Reviewer: ```suggestion
```

   :term:`SMUs <SMU>`, **24xx Standard**, :term:`PI`, ✅,
   , **24xx Interactive**, :term:`TSP`, ✅, ✅
   , **26xxB**, :term:`TSP`, ✅, 🚧

>    , **2636B**, :term:`TSP`, ✅, ✅

Reviewer: Is this separate from the 26xxB models?

Author: Other SMU models under 26XXB have not been fully qualified. We would need additional investigation to determine the common APIs between these listed models. Given the full qualification on available SMU2636B we would want to make SMU2636B as fully qualified instrument.
